src/main/java/pro/beam/api/futures/checkers/Tetris.java

```diff
@@ -0,0 +1,20 @@
+package pro.beam.api.futures.checkers;
+
+import com.google.common.collect.ImmutableMap;
+import pro.beam.api.exceptions.BeamException;
+import pro.beam.api.exceptions.tetris.MissingGameException;
+import pro.beam.api.futures.SimpleFutureChecker;
+import pro.beam.api.resource.tetris.RobotInfo;
+
+public class Tetris {
+    private static final int GAME_UNSET_CODE = 403;
```

Comment:
Perhaps better use `GAME_NOT_SET_CODE` as I initially assumed this was the code returned when the game has been unset (Like removed), and it didn't make sense.